README.md

```diff
@@ -1 +1,137 @@
-# To-do-List
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+    <br/>
+
+  <h3><b>To do list README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+> Single page application that allows users to keep track of their tasks.
+
+**[To do List]** is a single page application that allows users to keep track of their tasks. Users can add the task and also check the checkbox if the task has been completed.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML5
+> CSS
+> Javascript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[User can add task]**
+- **[User can mark complete task]**
+- **[User can delete completed tasks]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Page can be viewed by opening the index.html on your browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+> Install Git in your computer Git
+> Any text editor VSCode
+> Install linters on your local env't
+
+### Setup
+
+Clone this repository to your desired folder:
+
+ git clone https://github.com/tjay1760/To-do-List.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **John Thiongo**
+
+- GitHub: [@githubhandle](https://github.com/tjay1760)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```

Comment:
[OPTIONAL] Kindly remove your Twitter link if you don't have one. The README without broken link will look more professional.